src/main/steps/applicant1/hub-page/joint/latest-update-content/clarification-submitted.njk

```diff
@@ -0,0 +1,12 @@
+{% if isClarificationResponsesOrDocumentsUploaded %}
```

Comment:
would it not be better to use cannotUploadDocuments and else instead of the else if?

Reply:
Updated statement to only include cannotUploadDocuments.